### PR TITLE
Python3 fixup

### PR DESCRIPTION
**Pull request by sathnaga**
Let's port wrapper  and pci_info to work only on python3 and do respective update in README.

### CODE REVIEW

**Comment by sathnaga** [comment]
```
RHEL77:
# rpm -q python3-devel
python3-devel-3.6.8-10.el7.ppc64le
# rpm -q python3-setuptools
python3-setuptools-39.2.0-10.el7.noarch
#

RHEL81:
# rpm -q python36-devel
python36-devel-*
# rpm -q python3-setuptools
python3-setuptools-*

Fedora28:
# rpm -q python3-devel
python3-devel-3.6.8-3.fc28.ppc64le
# rpm -q python3-setuptools
python3-setuptools-39.0.1-1.fc28.noarch

Ubuntu 1804:
# dpkg -l|grep ' python3-dev'
ii  python3-dev                            3.6.7-1~18.04                          ppc64el      header files and a static library for Python (default)
# dpkg -l|grep ' python3-setuptools'
ii  python3-setuptools                     39.0.1-2                               all          Python3 Distutils Enhancements

SLES15SP1:
# rpm -q python3-devel
python3-devel-3.6.8-3.23.1.ppc64le
# rpm -q python3-setuptools
python3-setuptools-40.5.0-4.21.noarch


SLES12SP5:
# rpm -q python3-setuptools
python3-setuptools-18.0.1-4.8.1.noarch
# rpm -q python3-devel
python3-devel-3.4.6-25.29.1.ppc64le```

File: README [hunk 1]
```diff
@@ -222,63 +230,64 @@
...
       You must be a proficient/advanced Avocado test suite user to correctly customize the Guest sanity suites since the syntax of these
       configuration files is very specific.
-   
+
    Adding Host Sanity Tests:
-      The Host sanity tests are listed in the config/tests/host/sanity.cfg file in this repository.  These are tests used directly 
-      from the avocado-misc-tests respository (see link in the REFERNCE LINKS section below).  If you want to run additional 
-      tests from this repository, simply add a new line in the sanity.cfg file with the location of the file within the 
+      The Host sanity tests are listed in the config/tests/host/sanity.cfg file in this repository.  These are tests used directly
+      from the avocado-misc-tests respository (see link in the REFERNCE LINKS section below).  If you want to run additional
```
Comment (balamuruhans) [inline README]: `%s/REFERNCE/REFERENCE`

**Comment by sathnaga** [comment]
python 3.7.4
```
./avocado-setup.py --bootstrap --enable-kvm
./avocado-setup.py:34: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  CONFIGFILE = configparser.SafeConfigParser()
./avocado-setup.py:36: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  NORUNTESTFILE = configparser.SafeConfigParser()
./avocado-setup.py:38: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  INPUTFILE = configparser.SafeConfigParser()
15:51:23 INFO    : Check for environment
15:51:23 INFO    : Creating Avocado Config
15:51:27 INFO    : Bootstrapping Avocado
15:51:27 INFO    : Cloning the repo: avocado in /home/bala/tests/tests/avocado
15:51:30 INFO    : Installing avocado from /home/bala/tests/tests/avocado
15:52:19 INFO    : Cloning the repo: avocado-vt in /home/bala/tests/tests/avocado-vt
15:52:25 INFO    : Installing avocado-vt from /home/bala/tests/tests/avocado-vt
15:52:33 INFO    : Bootstrapping vt libvirt
15:52:48 INFO    : Bootstrapping vt qemu
15:52:53 INFO    : Cloning the repo: avocado-misc-tests in /home/bala/tests/tests/tests/avocado-misc-tests
15:52:55 INFO    : Downloading the guest os image
15:54:42 WARNING : Avocado html plugin not installed
15:54:42 INFO    : Installing optional plugin: html
15:54:46 WARNING : Avocado varianter_yaml_to_mux plugin not installed
15:54:46 INFO    : Installing optional plugin: varianter_yaml_to_mux
15:54:46 INFO    : Removing temporary mux dir

# echo $?
0

```

python 3.6.8
```
# ./avocado-setup.py --bootstrap --enable-kvm
06:16:06 INFO    : Check for environment
06:16:06 INFO    : Creating temporary mux dir
06:16:06 INFO    : Creating Avocado Config
06:16:10 INFO    : Bootstrapping Avocado
06:16:10 INFO    : Updating the repo: avocado in /home/sath/tests/avocado
06:16:11 INFO    : Installing avocado from /home/sath/tests/avocado
06:16:18 INFO    : Cloning the repo: avocado-vt in /home/sath/tests/avocado-vt
06:16:25 INFO    : Installing avocado-vt from /home/sath/tests/avocado-vt
06:16:57 INFO    : Bootstrapping vt libvirt
06:18:21 INFO    : Bootstrapping vt qemu
06:18:25 INFO    : Updating the repo: avocado-misc-tests in /home/sath/tests/tests/avocado-misc-tests
06:18:26 INFO    : Downloading the guest os image
06:20:51 WARNING : Avocado varianter_yaml_to_mux plugin not installed
06:20:51 INFO    : Installing optional plugin: varianter_yaml_to_mux
06:20:51 INFO    : Removing temporary mux dir

# echo $?
0
```

File: config/wrapper/env.conf [hunk 2]
```diff
@@ -4,131 +4,131 @@ avocado_vt = https://github.com/avocado-framework/avocado-vt.git
...
+packages = gcc,python3-devel,xz-devel,python3-setuptools,numactl,policycoreutils-python
 [deps_rhel7_NV]
 packages =
 [deps_rhel7_pHyp]
 packages =
 [deps_rhel7_kvm]
 packages = p7zip,libvirt-devel,tcpdump,virt-install,qemu,libvirt,SLOF,genisoimage,attr
 [deps_rhel8]
-packages = gcc,python2-devel,xz-devel,python2-setuptools,numactl,policycoreutils-python-utils
+packages = gcc,python36-devel,xz-devel,python3-setuptools,numactl,policycoreutils-python-utils
```
Comment (narasimhan-v) [inline config/wrapper/env.conf]: python3 or python36 ?

File: lib/pci.py [hunk 3]
```diff
@@ -19,9 +19,10 @@
 
 import re
 import os
-import commands
 import platform
 
+from .helper import runcmd
```
Comment (narasimhan-v) [inline lib/pci.py]: Should we integrate pci and wrapper for now ?
The reason this was not done is to have it separated whenever needed.

File: config/wrapper/env.conf (hunk 2 above)
Comment (sathnaga) [reply config/wrapper/env.conf]: its python36

File: lib/pci.py (hunk 3 above)
Comment (sathnaga) [reply lib/pci.py]: not getting you, you mean to move all methods in pci.py to helper.py?

Comment (sathnaga) [reply lib/pci.py]: it is better to keep this way, as sooner or later we need this piece to be out of wrapper right....

Comment (narasimhan-v) [reply lib/pci.py]: Yes, so should it use runcmd ?

Comment (sathnaga) [reply lib/pci.py]: yep, till it resides here, its fine, and when we decide to move somewhere else, it can carry `runcmd` defintion aswell..

**Pull request merged**